app/views/provider_interface/application_data_export/new.html.erb

```diff
@@ -45,7 +39,11 @@
         <%= f.govuk_collection_check_boxes :provider_ids, @application_data_export_form.providers_that_actor_belongs_to, ->(p) { p.id.to_s }, :name, legend: { text: 'Select applications for certain organisations', size: 'm' } %>
       <% end %>
 
-      <%= f.govuk_submit 'Export data (CSV)' %>
+      <div class="govuk-inset-text">
+        Sex, disability and ethnicity information will be marked as confidential if you do not have permission to view it, or if the candidate has not accepted your offer.
+      </div>
+
+      <%= f.govuk_submit 'Export application data (CSV)' %>
```

Comment:
```suggestion
      <%= f.govuk_submit 'Export data' %>
```

(per screenshot on card)

Reply:
I didn't update the screenshot but I added a note on the card as there was a discussion following that: https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1628591398080000

Reply:
ok, great — please disregard in that case!